config.yaml

```diff
@@ -107,6 +107,9 @@ synced_folders:
 # Values: 5.6, 7.0, 7.1, 7.2, 7.3 (or 5.6.30)
 php: 7.3
 
+# Maximum file upload size. This will set post_max_size and upload_max_filesize in PHP and client_max_body_size in Nginx.
+upload_size: 1024M
```

Comment:
I wonder whether we should have done
```yaml
php:
    version: 7.3
    upload_size: 1024M
```

instead. Pain to change now, but in hindsight...

Reply:
I thought about that but it would mean I'd have to update all the extensions for that and that would take ages!